### ISSUE TITLE
Fastlane setup

### ISSUE THREAD

**Issue by kostiakoval** [agensdev/playbook-ios#8]
Would be awesome to have a common setup and functions that we use be located in 1 place Like:
- jira integration
- actions for feature, release, test
- additional helpers functions

**Comment by hfossli** [comment]
Yep! Continue the discussion here about what you guys want and I'll book an ios dev for making this happen company wide.

**Comment by hfossli** [comment]
I think we should have a sample repo/project (either in this repo or separate) which shows how to do fastlane, carthage, project structure and more.

**Comment by kostiakoval** [comment]
Agree we should definitely have a sample project. The question is what should in include? It could serve as an example or as a template that you could copy when making new project or library. It could include many things:

- tools setup (fastlane, carthage)
- the coding style
- best practice 
- architecture 
- project setup
